VERSION

```diff
@@ -0,0 +1 @@
+1.0.0-pre1
```

Comment:
IIUC, this would be to differentiate the version of vector-datasource from the api version. There can be changes that sit outside the software that could affect the api.

We could just track the api version separately in the stack settings. I can see it being confusing that there are 2 separate version specifiers in the same repository.